File: Ejercicios de Semana Santa/ejercicios de semana santa php/Four php/Library/LicenciaC.php
<?php
require_once 'Licencia.php';

class LicenciaC extends Licencia {
    public function __construct() {
        $this->costo = 1600000;
        $this->duracion = 12;
    }

    public function mostrarServicios() {
        echo "Licencia C: vehículos de servicio público.\n";
    }
}
?>

File: Prueba y error/ejercicios de semana santa java/four java/Library/LicenciaC.java
package Library;

import java.text.DecimalFormat;

public class LicenciaC extends Licencia {
    public LicenciaC() {
        this.tipo = "LicenciaC (Vehículos particulares)";
        this.duracion = 36;  // Duración en meses
        this.costo = 650000;  //  (COP)
    }

    @Override
    public String detallesLicencia() {
        // Formatear el costo con punto y coma para miles y decimales
        DecimalFormat formato = new DecimalFormat("#,###.00");
        String costoFormateado = formato.format(costo);
        return "Tipo de Licencia: " + tipo + "\nDuración: " + duracion + " meses\nCosto: $" + costoFormateado + " COP";
    }
}